network/discovery/service_test.go

```diff
@@ -289,35 +280,11 @@ func TestDiscV5Service_checkPeer(t *testing.T) {
 
 func TestDiscV5ServiceListenerType(t *testing.T) {
 
-	t.Run("Post-Fork", func(t *testing.T) {
-		netConfig := PostForkNetworkConfig()
+	t.Run("Listener Type", func(t *testing.T) {
```

Comment:
We don't need to keep it inside a subtest as it's just one case. Also, the test name is a bit misleading because we don't have different listener types anymore